src/content/docs/usage/Node Graph/Tutorials/your-first-graph.mdx

```diff
@@ -11,69 +11,94 @@ import { Steps } from '@astrojs/starlight/components';
-- **Numbers** - Various numerical related nodes, most notably the Math node, which allows you to perform mathematical operations on numbers.
-- **Effects** - Standalone effects such as Raster Outline or custom Shader
-- **Workspace** - Nodes that allow for workspace cusomization - multiple outputs and export zones.
-- **Shape** - Nodes that allow you to create and manipulate shapes
-- **Matrix** - Matrix related nodes, such as Transform, which allow you to manipulate the position, rotation, and scale of nodes.
-- **Filters** - Nodes that contain various filters - for example, Blur, Shadow, and Color Correction.
-- **Animation** - Nodes that allow you to create procedural animations.
-- **Misc** - Miscellaneous nodes that don't fit into any other category, such as Document Info.
+---
 
-</details>
+### Example: Add a Shadow Filter node 
 
+Let’s add a Shadow Filter node and connect it to the Base Layer: 
 
 <Steps>
+1.  Go to the **Filters** category and click **Shadow Filter** to add it to the node graph.  
+2. Find the red circle <span style={{color: "#cc5c5c"}}>⬤</span> on the right side of the **Filter node** label.
+3. Drag the circle to the matching red socket on the Base Layer node (on the left side). 
+4. Select **Base Layer** in the Layers panel.
+5. Draw on your document, you should see a shadow effect applied in real-time.
 
-4. Let's add a <span class="pixi-icon small icon-cone"/> **Shadow Filter** node. You can find it in the **Filters** category. Click on it to add it to the Node Graph.
+</Steps>
 
-5. Click on the red-ish circle <span style={{color: "#cc5c5c"}}>⬤</span>, next to the "Filter" label, on the right side of the <span class="pixi-icon small icon-cone"/> **Shadow Filter** node, and drag it to the "Filter" socket with the same color in the **<span class="pixi-icon small icon-layers-double"/> Base Layer** node (on the **left** side of the node). 
 
-6. Now go back to your document (make sure Base Layer is selected in the Layers panel) and try drawing something. You should see a shadow effect applied to your drawing!
 
-</Steps>
+---
+
+## Types of nodes
+
+PixiEditor 2.0.0.97 includes several node categories: 
+
+- Color - Includes nodes that work with colors.
+- Image - Nodes such as create, mask, and those that manipulate images such as Noise, Mask.
+- Structure - Represent elements in a layer tree like Image Layers, Vector Layers, and Folders.
+- Numbers - Nodes that perform numerical operations. For example, Math node.  
+- Effects - Apply visual effects such as Raster Outline or Custom Shader. 
+- Workspace - Nodes for customizing outputs and exporting zones.
+- Shape - Nodes allow you to create and manipulate shapes.
+- Matrix - Matrix-related nodes like Transform.
+- Filters - Apply effects like Blur, Shadow, and Color Correction with these nodes.
+- Animation - Nodes for creating procedural animations.
+- Misc - Miscellaneous nodes, such as Document Info.
```

Comment:
I'd keep bold category names. Imo it's easier to read

<img width="816" height="648" alt="Image" src="https://github.com/user-attachments/assets/0c09d657-30bd-4e6a-bb1c-07db05ffa390" />